app/src/main/res/values-en/strings.xml

```diff
@@ -1122,15 +1122,15 @@ If there are no signs along the whole street which apply for the highlighted sec
 
     <string name="quest_policeType_title">Which type of police station is this?</string>
 
-    <string name="quest_postboxCollectionTimes_title">What are the collection times of this postbox?</string>
-    <string name="quest_postboxCollectionTimes_resurvey_title">Are the collection times of this postbox still correct?</string>
+    <string name="quest_postboxCollectionTimes_title">What are the collection times of this mailbox?</string>
+    <string name="quest_postboxCollectionTimes_resurvey_title">Are the collection times of this mailbox still correct?</string>
     <string name="quest_collectionTimes_add_times">Add collection time</string>
     <string name="quest_collectionTimes_answer_no_times_specified">No times specified</string>
 
-    <string name="quest_postboxRef_title">What’s the reference number of this postbox?</string>
-    <string name="quest_ref_answer_noRef">"No reference number visible…"</string>
+    <string name="quest_postboxRef_title">What’s the number of this mailbox?</string>
```

Comment:
is it possible to have number on box which is not a reference code? I think so.

Reply:
I have no idea what a "reference code" is and that term sounds made-up to me.  "What's the number of" implies that the mailbox is one of a series of numbered mailboxes.

Reply:
There can be a lot of numbers on a postbox - postcode, telephone number, house number, (collection times, ...)

The `ref` on a postbox really is just an ID provided by the operator. It does not actually need to be a number at all. Such code/number is usually shown in context of a message like "call this phone number if this postbox has been damaged, supply this number/code: XYZ".

Reply:
"identification number" would be far better than "reference number" IMO

Reply:
iD steers clear of the term “reference” because it’s jargony enough that users don’t realize it’s just “the number” (which may be alphanumeric).

I’m not sure if collection boxes in the U.S. even have posted codes, but if they do then they’d probably be called “identification numbers” or somesuch, and they’d only be for internal maintenance purposes. Otherwise, mail carriers probably refer to them by the nearest address, like the public does.

Reply:
Just updated to "identification number" which I agree is more clear.  I'm also not sure if these actually exist but I'll check the next time I'm out and about ;)

Reply:
reference is like osmism from `ref` tag (or maybe UKism?)